src/components/Select/ButtonSelect/ButtonSelect.tsx

```diff
@@ -107,7 +107,7 @@ export const ButtonSelect = (props: TButtonSelectPropTypes): ReactElement => {
                   OptionRightIconComponent={optionRightIconComponent}
                   LabelRightIconComponent={labelRightIconComponent}
                   avatar={avatar}
-                  disabled={item.disabled}
+                  disabled={item.disabled || isSelected}
```

Comment:
@Hakob1504 jan i will suggest to do it from project side. there is case that for example you want to unselect, but incase if it is disabled. you can not